src/test/scala/io/dockstore/Requests.scala

```diff
@@ -0,0 +1,482 @@
+package io.dockstore
```

Comment:
Any tips on how to keep this (or other classes) up-to-date? 
Is there a structure to this (i.e. add this here, and then add this matching section here, etc.)?